modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -816,7 +816,9 @@
 
     /**
      * WAL rebalance threshold.
+     * @deprecated use Distributed MetaStorage property {@code wal.rebalance.threshold}.
```

Comment:
I think it should be corrected in accordance with GridCacheDatabaseSharedManager#HISTORICAL_REBALANCE_THRESHOLD_DMS_KEY